src/style.css

```diff
@@ -1 +1,77 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  color: #000;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  background-color: #fff;
+}
+
+.shell {
+  max-width: 1200px;
+  width: 100%;
+  margin: 0 auto;
+  padding: 0 50px;
+}
+
+.header__shell {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
```

Comment:
```suggestion
.nav__list {
```

Reply:
Better to make names more clear, at first I thought to write comments that navs don't have ant default margins or paddings 🙂